Makefile_cc65

```diff
@@ -32,21 +37,20 @@ LIB_OBJECTS = \
 	src/tests.o \
 	src/time.o
 
-.PHONY: all clean cleanall
+.PHONY: all clean cleanall test
 
-all: libmega65.a
+libmega65.a: $(LIB_OBJECTS)
+	$(AR65) a $@ $^
+
+all: libmega65.a $(TESTS_PRGS)
```

Comment:
all needs to be the first valid target so that make without a target will pick all. With this in place, libmega65.a will be the primary target.
So please put all as the first target, except you had a recent for changing the order.

Reply:
Fixed.